docs/keywords.json

```diff
@@ -9186,5 +9186,752 @@
     ],
     "category":"Wildlife",
     "release":"2.1.5"
+  },
+  "coyote":{
+    "alias":[
+      "2.1.5",
+      "wildlife",
+      "animal,mammal,nature"
```

Comment:
It looks like your list of tags for each symbol got pasted as one tag in a single line because they were entered without spaces after each comma (e.g., `tags: animal,mammal,nature` verses `tags: animal, mammal, nature`)

There are also seems to be 18 concepts that already had a set of tags in this keywords.json

<img width="492" alt="image" src="https://user-images.githubusercontent.com/29716057/164344629-4f748506-02b6-4e90-ac9a-603cbfd51788.png">